integration_tests/integration/allocate-to-practitioner.spec.ts

```diff
@@ -54,19 +54,12 @@ context('Allocate to Practitioner', () => {
     })
     choosePractitionerPage.allocateCaseButton().click()
     const allocatePage = Page.verifyOnPage(AllocateToPractitionerPage)
-    allocatePage.subHeading().should('have.text', "You're allocating this case to probation practitioner John Doe (PO)")
+    allocatePage.headingText().should('have.text', "\n    You're allocating Dylan Adam Armstrong (C1) to John Doe (PO)")
```

Comment:
Think it would be better to change this to `.should('contain', "You're allocating this case to probation practitioner John Doe (PO)")` rather than testing for a line break

Reply:
fixed